Octokit.Reactive/Clients/ObservableTeamDiscussionsClient.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Reactive;
+using System.Reactive.Threading.Tasks;
+using Octokit.Reactive.Internal;
+
+namespace Octokit.Reactive
+{
+    /// <summary>
+    /// A client for GitHub's Team Discussions API.
+    /// </summary>
+    /// <remarks>
+    /// See the <a href="https://developer.github.com/v3/teams/discussions/">Team Discussions API documentation</a> for more information.
+    /// </remarks>
+    public class ObservableTeamDiscussionsClient : IObservableTeamDiscussionsClient
+    {
+        private static readonly string AcceptHeader = AcceptHeaders.Concat(AcceptHeaders.TeamDiscussionsApiPreview, AcceptHeaders.ReactionsPreview);
+
+        readonly IConnection _connection;
+        readonly ITeamDiscussionsClient _client;
+
+        /// <summary>
+        /// Initializes a new Organization Teams API client.
+        /// </summary>
+        /// <param name="client">An <see cref="IGitHubClient" /> used to make the requests</param>
+        public ObservableTeamDiscussionsClient(IGitHubClient client)
+        {
+            Ensure.ArgumentNotNull(client, nameof(client));
+            _connection = client.Connection;
+            _client = client.Organization.TeamDiscussions;
+        }
+
+        public IObservable<TeamDiscussion> Get(int teamId, int number)
```

Comment:
XmlComments missing for all the methods in this class (just copy the ones from interface)